core/src/test/scala/unit/kafka/server/ReplicationQuotasTest.scala

```diff
@@ -130,20 +131,15 @@ class ReplicationQuotasTest extends ZooKeeperTestHarness {
     val start = System.currentTimeMillis()
 
     //When we create the 2 new, empty brokers
-    brokers = brokers :+ TestUtils.createServer(fromProps(createBrokerConfig(106, zkConnect)))
-    brokers = brokers :+ TestUtils.createServer(fromProps(createBrokerConfig(107, zkConnect)))
+    createBrokers(106 to 107)
```

Comment:
A few lines above you call `get` on each future as you send the data. Is that required for the test? Otherwise you may prefer to return the `Future` and after all the `send`s are done, you can `foreach(_.get)`.